cpp/data/NewsletterUploadForm.html

```diff
@@ -0,0 +1,14 @@
+<html>
+  <head>
+    <title>Newsletter Upload Form</title>
+  </head>
+  <body>
+    The uploaded file must be a plin-text file.  The first line will be used as the subject of the
+    email message and the remainder as the message body.  The encoding of the file must be UTF-8 with
+    no leading BOM.
+    <form enctype="multipart/form-data" action="/var/www/cgi-bin/newsletter_uploader" method="post">
```

Comment:
I still doubt this works, but still => shouldn't the .py extension be added to the script name?

Reply:
I found several examples that support that a bare path works.  I have added the missing extension.